### ISSUE TITLE
UCSD Ped2

### ISSUE THREAD

**Issue by joos2010kj** [jx-zhong-for-academic-purpose/GCN-Anomaly-Detection#28]
Hello! Can I ask two questions?

1) how did you **randomly select** the videos for the train set, as there is no standard for **randomness** mentioned? Like, did you use numpy.random.seed(0)? If possible, could you provide a txt containing 60 anomaly videos and 40 normal videos used (6 abn * 10 + 4norm * 10) for the train?

2) So, is the training set just composed of 6 random abnormal videos and 4 random normal videos from UCSD Ped2? And the remaining 18 videos of abnormal and normal videos are used as test set?

I just keep having doubts about how the model can be trained on such a tiny data and have such an excellent performance b/c each of those 10 "videos" has between 120 and 180 frames only.

Thank you!

**Comment by zimengxueying** [comment]
> 

Hi！How you ever solved he problem?
